Code_of_Conduct.md

```diff
@@ -0,0 +1,65 @@
+# Code Of Conduct for ALiAS
+
+Code of Conduct guidelines for Amity Linux Assistance Sapience (AliAS)!
+
+## Be respectful
+
+
+Respect yourself, and respect others. Be courteous to those around you. If someone indicates they don’t wish to be photographed, respect that wishes. If someone indicates they would like to be left alone, let them be. Our event venues and online spaces may be shared with members of the public; please be considerate to all patrons of these locations.
+
+Before Direct Messaging someone, ask for their permission on the group. If they allow you to, only then proceed to do so. The people in here are here to learn, explore and socialize in a professional environment.
+
+## Be inclusive
+
+
+All presentation material should be suitable for people aged 12 and above.
+
+Any public presentation which is part of any event, including but not limited to keynotes, presentations, lightning talks, addresses, mailing list posts, telegram posts and forums or any other form of communication that directly or indirectly indicates the involvement of the community, is subject to this code of conduct and thus may not contain:
+
+    - sexual or violent imagery;
+    - exclusionary language;
+    - harassment and sexist, racist, or exclusionary jokes;
+    - insults or ad-hominem attacks.
+
+Presenters are asked to avoid language which is not appropriate for an all-ages audience as much as possible. Please use English wherever possible.
+
+If the subject matter of the presentation cannot be presented adequately without including language that could be considered offensive, this should be pointed out in advance, at the beginning of the talk and in the schedule. Failing to do so, will lead to actions mentioned in ["What does that mean for me?"](#What-does-that-mean-for-me?) section.
+
+If presenters are unsure whether their material is suitable, they are encouraged to show it to the event’s speaker liaison or send the same for reviews to admins before their session.
+
+## Be aware.
+
+
+We ask everyone to be aware that we will not tolerate intimidation, harassment, or any abusive, discriminatory or derogatory behaviour by anyone at any event or online.
+
+Complaints can be made to the admins or teacher representative of the group meeting them in-person contacting them through direct messages. All complaints made to event admins will remain confidential and be taken seriously. The complaint will be treated appropriately with discretion. Should admins or teacher representative consider it appropriate, measures they may take include:
+
+    - the individual(s) may be told to apologise
```

Comment:
Please add punctuations at the end.
The other blocks (above and below) have it, hence it'd be good to have it here, too.
Same applies to all the points below :)